LayoutTest/TestCase/LYTLayoutFailingTestSnapshotRecorder.m

```diff
@@ -69,12 +74,29 @@ - (instancetype)init {
     return self;
 }
 
+- (void)testSuiteWillStart:(XCTestSuite *)testSuite {
+    self.failedTestsDuringTestSuite = 0;
+}
+
+- (void)testSuiteDidFinish:(XCTestSuite *)testSuite {
+    if (self.failedTestsDuringTestSuite > 0 && self.failingTestsSnapshotFolders.count > 0) {
+        SimpleLog(@"\nSnapshots of failing tests can be found in:\n");
+        for (NSString *path in self.failingTestsSnapshotFolders) {
+            SimpleLog(@"%@\n", path);
+        }
+        SimpleLog(@"\n");
+    }
```

Comment:
do you want to clear out failingTestsSnapshotFolders here like you do in testBundleDidFinish?